packages/slate-editor/src/extensions/blink-nodes/components/BlinkNodes.tsx

```diff
@@ -0,0 +1,67 @@
+import type { RefObject } from 'react';
+import React, { useEffect, useState } from 'react';
+import type { Editor, Node } from 'slate';
+import { ReactEditor, useSlateStatic } from 'slate-react';
+
+import styles from './BlinkNodes.module.scss';
+
+export function BlinkNodes({ containerRef }: { containerRef: RefObject<HTMLDivElement> }) {
+    const editor = useSlateStatic();
+
+    return (
+        <>
+            {editor.nodesToBlink.map(([top, bottom], index) => (
+                <Blinker
+                    key={index}
+                    editor={editor}
+                    top={top}
+                    containerRef={containerRef}
+                    bottom={bottom}
+                    clear={() =>
+                        (editor.nodesToBlink = editor.nodesToBlink.filter((_, i) => i !== index))
```

Comment:
What if there's another blink pair added while this one is still animating?
Wouldn't it be better to check for object identity instead?

```ts
editor.nodesToBlink = editor.nodesToBlink.filter((pair) => pair[0] === top && pair[1] === bottom)
```
?